tic_tac_toe.rb

```diff
@@ -0,0 +1,89 @@
+class Grid
```

Comment:
Let's iterate a bit on your `Grid` class. A simple rule of thumb for organization of your code is to try and name what a certain unit (for example, a class) does. If, while naming it, you say "and", it does too much".

Your `Grid` class stores the state of the `Tic-Tac-Toe` game, is concerned with rendering it to the screen *and* implements the coordination of game progress. I invite you to reorganise the logic in the following way:
- your grid class should not `puts` anything. It should simply store it's internal state and allow updating it.
- You should have a separate entity managing the flow of the game. Maybe a `Game` class?

On top of that, I'd like you to start thinking about your files and folders structure: each unit should have a dedicated file. In your app, those could be `grid.rb` and `game.rb`.